tests/Application/LaravelConfigExtendingTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Application;
+
+use NunoMaduro\PhpInsights\Application\Adapters\Laravel\Preset;
+use NunoMaduro\PhpInsights\Application\ConfigResolver;
+use NunoMaduro\PhpInsights\Domain\Insights\ForbiddenDefineGlobalConstants;
+use NunoMaduro\PhpInsights\Domain\Insights\ForbiddenPrivateMethods;
+use PHP_CodeSniffer\Standards\Generic\Sniffs\PHP\ForbiddenFunctionsSniff;
+use PHPUnit\Framework\TestCase;
+
+final class LaravelConfigExtendingTest extends TestCase
+{
+    /**
+     * @var array
+     */
+    private $laravelPresetConfig;
+
+    /**
+     * @var array
+     */
+    private $configExtending;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->laravelPresetConfig = Preset::get();
+        $this->configExtending = require __DIR__ . '/../Fixtures/ConfigFiles/Laravel/config-extending.php';
+    }
+
+    public function testExtendingConfigKey_Exclude(): void
```

Comment:
Use camel case through the whole method name

Reply:
Sure.

(that was for capitalizing in shell output) first time i did it :)